tryolabs/santiago-challenge-tryolabs-latam
# Version 0.1 merge (#9)

* Add gitignore

* Add packaging files

* Add formatting to python script files

* Fixes to make notebook work

* Notebook review, challange updated.

* Fix data loading directory for test

* update gitignore

* fix prediction test

* Add comment to challange.md

* Passed models tests

* Add poetry lock file

* Fix model bugs

* Additional model improvements

* Remove ipdb from model training.

* Update fastapi version

* Add api

* Update gitignore

* Removed unused imports

* Update model storing directory

* Chande model loading

* Change directory for storing the model

* Update poetry with appropriate versions

* Add dockerfile

* Update dockerfile command

* Add deploy script, and docker fix

* update library version for stress test

* update stress tests url

* Update doc

* Add copy of model to dockerfile

* Update dockerfile and poetry lock

* Add change to dockerfile

* Add training to deploy script

* Moved ci/cd files to appropriate directory.

* First ci update

* Add paths to testing

* Add .github folder to trigger

* add '' to paths

* Add commands to ci

* Fix syntax error

* Add module files to needed paths

* setup python version

* Check directory

* Checkout code

* Fix installation

* add virtualenv folder to executing machine

* Add test dependencies to poetry

* Modified dependencies installation

* Add model retrival

* Add api test

* rename job

* Add cd workflow

* Fix syntax error

* Check secret upload

* Add gcp authentication

* Add github actions workflows to PRs on main.
